sdk/storage/examples/list_containers2.rs

```diff
@@ -0,0 +1,71 @@
+use azure_core::HttpClient;
+use azure_storage::clients::*;
+use serde::Serialize;
+use std::error::Error;
+use std::sync::Arc;
+
+#[derive(Debug, Clone, Serialize)]
+struct SampleEntity {
```

Comment:
Get rid of the individual attributes and use `#[serde(rename_all = "snake_case")]`